storage/src/backend/connection.rs

```diff
@@ -215,12 +220,23 @@ impl Connection {
                         let client = Client::new();
                         let _ = client
                             .get(ping_url.clone())
-                            .timeout(Duration::from_secs(connect_timeout))
+                            .timeout(Duration::from_secs(connect_timeout as u64))
                             .send()
                             .map(|resp| {
-                                proxy.health.set(is_success_status(resp.status()));
+                                let success = is_success_status(resp.status());
+                                if !success {
+                                    warn!(
```

Comment:
Is it still possible to print a lot of logs when the proxy server is consistently unhealthy? I think maybe only print the log when the health status changed.

Reply:
good idea